packages/fx-core/src/common/local/constants.ts

```diff
@@ -145,7 +145,7 @@ export const TaskDefaultValue = Object.freeze({
     },
   },
   npmInstall: {
-    npmInstallArgs: ["--no-audit"],
+    npmInstallArgs: ["--no-audit", "--legacy-peer-deps"],
```

Comment:
Will this cause another issue: `teamsfx` requires `teams-js 2.0`, but user adds `teams-js 1.0`, then tab (SSO) does not work.

Reply:
No. After testing out, it works properly, since teams-js 2.0 still provides an API translation layer (mapping v.1 SDK to v.2 SDK API calls).